src/ecosystem/Ecosystem.jl

```diff
@@ -106,6 +106,8 @@ Base.@kwdef struct Criteria{P,N} <: EcoModel
         name="Low Area Tolerance", description="Tolerance for low proportional space for seeding deployments.")
     deployed_coral_risk_tol::P = Param(1.0, ptype="real", bounds=(0.75, 1.0), dists="unif",
         name="Risk Tolerance", description="Filters out sites with heat/wave stress above threshold.")
+    use_dist::N = Param(0, ptype="integer", bounds=(0, 1 + 1), dists="unif",
+        name="Use Distance", description="Turns using distance sorting on and off.")
```

Comment:
How about "Distance Threshold" as the name?

For the description, how about: "Turns ~using~ distance sorting on **or** off"

Reply:
Sure, sounds good. Are these used for presenting analysis in a more understandable format?

Reply:
Yes

Reply:
Ok, cool